rules.mk.j2

```diff
@@ -98,7 +98,8 @@ deploy: package
 	$(Q) if [[ $${DRONE_BUILD_EVENT:-} == "push" ]] && [[ $${DRONE_BRANCH:-} == "master" ]]; then mc cp s3/tinkerbell-oss/osie-uploads/$v.tar.gz s3/tinkerbell-oss/osie-uploads/latest.tar.gz; fi
 	$(E) "UPLOAD   s3/tinkerbell-oss/osie-uploads/$v.tar.gz.sha512sum"
 	$(Q) mc cp build/$v.tar.gz.sha512sum s3/${s3Bucket}/osie-uploads/$v.tar.gz.sha512sum
-	$(Q) if [[ $${DRONE_BUILD_EVENT:-} == "push" ]] && [[ $${DRONE_BRANCH:-} == "master" ]]; then mc cp s3/tinkerbell-oss/osie-uploads/$v.tar.gz.sha512sum s3/tinkerbell-oss/osie-uploads/latest.tar.gz.sha512sum; fi
+	$(Q) if [[ $${DRONE_BUILD_EVENT:-} == "push" ]] && [[ $${DRONE_BRANCH:-} == "master" ]]; then echo "# $(cat build/$v.tar.gz.sha512sum)" > build/latest.tar.gz.sha512sum && echo "$(cat build/$v.tar.gz.sha512sum)" | sed 's| (.*) | (latest.tar.gz) |' >> build/latest.tar.gz.sha512sum; fi
```

Comment:
this can just be:
```
$(Q) if [[ $${DRONE_BUILD_EVENT:-} == "push" ]] && [[ $${DRONE_BRANCH:-} == "master" ]]; then sed 's| (.*) | (latest.tar.gz) |' build/$v.tar.gz.sha512sum >build/latest.tar.gz.sha512sum; fi
```
instead right?